docs/getstarted/core_concepts.md

```diff
@@ -33,9 +35,28 @@ model that meets your needs.
 
 On FloydHub, a project is a collection of all the work and iterations you
 perform when developing a deep learning model. In contrast to a typical deep
-learning workflow, your experiments and iterations (we call them
+learning workflow, your experiments and training iterations (we call them
 [jobs](#jobs)) will be versioned and kept organized for you to reference in the
-future.
+future. Workspaces provide interactive environments for you to prepare your 
+training jobs, including pre-processing, data exploration, and other activities.
+
+## Workspaces
+
+!!! tip "Quick Look"
+    A workspace is an interative development environment on FloydHub for model
+    development, pre-processing data, and other steps in your deep learning
+    workflow.
```

Comment:
Can you add a screenshot of the Workspace inside this tip?

Reply:
Sure! Will do